lib/teleterm/api/proto/v1/cluster.proto

```diff
@@ -43,8 +43,10 @@ message LoggedInUser {
   repeated string roles = 2;
   // ssh_logins is the user ssh logins
   repeated string ssh_logins = 3;
+  // active_requests is an array of request-id strings of active requests
+  repeated string active_requests = 4;
   // acl is the user acl
-  ACL acl = 4;
+  ACL acl = 5;
```

Comment:
The whole point of numbered fields is so that you can add/remove fields without breaking compatibility. By changing acl from 4 to 5 you've broken compatibility.

Granted, that may not matter in this case, since this GRPC interface is only between connect and its embedded tsh binary, but I still don't see a need to break things - we could just make `active_requests` field 5 instead.

Reply:
This was brought up previously when changing `ListServers` to `GetAllServers` as any older client would then have `ListServers` return an unimplemented error. You are correct in that it won't affect us here because of the embedded binary, but it's best to keep this practice and not lose its value. Thanks for pointing this one out. Will change active_requests to field 5 instead.